### PR TITLE
Sr packages form

### PR DESCRIPTION
**Pull request by seancrossettie**
Added working packages form

## Description
Added a form that adds a package card to the array of packages. 

## Related Issue
https://github.com/nss-evening-cohort-14/gitsub-e14-team-2-electric-boogaloo/issues/22

## Motivation and Context
Required for MVP.

## How Can This Be Tested?
Pull down this repo and run it on your local server.

## Screenshots (if appropriate):
Screen Shot 2021-02-10 at 3 17 09 PM

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,11 +15,10 @@
     <title>Gitsub</title>
   </head>
   <body>
-
     <nav class="navbar navbar-expand-lg navbar-dark">
       <ul class="navbar-nav">
         <li class="nav-item">
-          <a href="#" class="nav-link">Overview</a>
+          <a href="index.html" class="nav-link">Overview</a>
         </li>
```
Comment (hmparsons-sde) [inline index.html:22]: Shouldn't this link to overview.html?

Comment (seancrossettie) [reply index.html:22]: I was under the impression we were treating the index.html as the overview page.

Comment (seancrossettie) [reply index.html:22]: Unless I am mistaken!

**Review by hmparsons-sde — APPROVED** [review]

**Review by Gabrielle-Tobermann — APPROVED** [review]
Looks good! Only thing is, the projectsCards function was deleted - not sure if that will be a problem when merging.

**Review by hmparsons-sde — APPROVED** [review]

**Pull request merged**